components/home/featureSection1.tsx

```
@@ -0,0 +1,242 @@
+"use client"
+
+import type React from "react"
+import { motion } from "framer-motion"
+import {
+  Code,
+  Users,
+  Globe,
+  Infinity,
```

Biome (1.9.4)

[error] Do not shadow the global "Infinity" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)